theme/static/js/custom.js

```diff
@@ -222,23 +222,37 @@ $(document).ready(function () {
         });
     }
 
+    function popup_profile_alert(uid, fields) {
+        if ($("#publish").length) {
+                $("#publish").toggleClass("disabled", true);
```

Comment:
just being extra cautious here, can you confirm whitespace won't generate a length value > 0 or just quickly make sure the logic avoids a situation like that?

Reply:
Yes, I can confirm this has nothing to do with whitespaces. This ```#publish``` element is defined here as the publish icon button https://github.com/hydroshare/hydroshare/blob/97048f2d7e72375f0e6a143e74a5e004e144e2d4/theme/templates/resource-landing-page/top-right-buttons.html#L103. This is the old code which has been working for some time which I did not make any update but just moved it into a function so that it can take care of multiple missing required fields.